package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "aepp-calldata-js",
+  "name": "@aeternity/aepp-calldata",
   "version": "0.0.1",
   "description": "Aeternity data serialization library",
-  "main": "Encoder.js",
+  "main": "src/Encoder.js",
   "keywords": "aeternity, aesophia, fate, calldata, serialization, encoder, decoder",
```

Comment:
this isn't a change. but my IDE tells me that the list of keywords should be an array

Reply:
Fair point, somehow related to the docs anyway, fixed in 7d64be5